docs/Get started/FAQ.md

```diff
@@ -24,6 +24,10 @@ There is, however, a way to accomplish this even though the answer to the questi
 
 The update frequency for labels is configurable with the [—osquery_label_update_interval](https://fleetdm.com/docs/deploying/configuration#osquery-label-update-interval) flag (default 1 hour).
 
+### Can I modify built-in labels?
+
+While it is possible to modify built-in labels using `fleetctl` or the REST API, doing so is not recommended because it can lead to errors in the Fleet UI.
```

Comment:
Add a quick "for more info, see [link to issue]", something like that.

Reply:
@zhumo link has been added.